controllers/drplacementcontrol_controller.go

```diff
@@ -2178,7 +2182,10 @@ func (r *DRPlacementControlReconciler) ensureDRPCStatusConsistency(
 		dstCluster = drpc.Spec.FailoverCluster
 	}
 
-	progress, err := r.determineDRPCState(ctx, drpc, drPolicy, placementObj, dstCluster, log)
+	progress, msg, err := r.determineDRPCState(ctx, drpc, drPolicy, placementObj, dstCluster, log)
+
+	log.Info(msg)
```

Comment:
Why log here an not in determineDRPCState()? we pass the logger to the function so it can log whatever needed. Here we log something that it returned without any context.

Reply:
I do agree that logging just the msg here without any context will make it hard to read the log line. Willing to let it be addressed in a different PR though.

Reply:
Ok, let's do that in a different PR.